examples/ci/game/run_example.py

```diff
@@ -0,0 +1,165 @@
+import json
+import os
+import subprocess
+import platform
+
+
+def run(cmd, error=False, env_script=None, file_stdout=None):
+    if env_script is not None:
+        env = "{}.bat".format(env_script) if platform.system() == "Windows" else ". {}.sh".format(env_script)
+        cmd = "{} && {}".format(env, cmd)
+    # Used by tools/scm check_repo only (see if repo ok with status)
+    print("Running: {}".format(cmd))
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    out, err = process.communicate()
+    out = out.decode("utf-8")
+    err = err.decode("utf-8")
+    ret = process.returncode
+
+    if file_stdout:
+        open(file_stdout, "w").write(out)
+
+    output = err + out
+    if ret != 0 and not error:
+        raise Exception("Failed cmd: {}\n{}".format(cmd, output))
+    if ret == 0 and error:
+        raise Exception("Cmd succeded (failure expected): {}\n{}".format(cmd, output))
+    return output
+
+def replace(filepath, old, new):
+    content = open(filepath).read()
+    new_content = content.replace(old, new)
+    if new_content == content:
+        raise Exception("No replacement of '{}' happened".format(old))
+    open(filepath, "w").write(new_content)
+
+
+############### Part 1 ###################################
+# Just create a project with a cool dependency graph, and
+# and 2 consuming applications, everything with version ranges
+print("- Part 1: Setup the project initial state -")
+run('conan remove "*" -f')  # Make sure no packages from last run
```

Comment:
Also a good approach could be to setup a CONAN_HOME to run this, to keep things clean too.
I still need to figure out a few things, like uploading the packages, and the final build->develop package promotion in Artifactory, this would be related, maybe I need to figure out a couple of different CONAN_HOMEs to really mimic a distributed environment.